### DISCUSSION TITLE
PCL没有23w06a且启动器版本为最新

### DISCUSSION THREAD

**Discussion by MCyuzhouRS** [Meloong-Git/PCL#1225] [提问]
PCL没有23w06a但是启动器版本为最新
![image](https://user-images.githubusercontent.com/124948981/217985506-5e971ea1-8f3d-498b-8cc2-5711c90b8a0e.png)

**Comment by WForst-Breeze** [comment]
镜像源更新速度较慢，所以经常会出现这种情况（）如果您的网络环境较好，可以使用上面这位朋友的解决方案；如果网络环境不佳，可能需要等待一段时间后才能下载最新版本。
